pkg/crc/preflight/preflight_windows.go

```diff
@@ -40,7 +40,7 @@ var hypervPreflightChecks = [...]Check{
 		configKeySuffix:  "check-hyperv-switch",
 		checkDescription: "Checking if the Hyper-V virtual switch exist",
 		check:            checkIfHyperVVirtualSwitchExists,
-		fixDescription:   "Unable to perform Hyper-V administrative commands. Please make sure to re-login or reboot your system",
+		fixDescription:   "Unable to perform Hyper-V administrative commands. Please reboot your system and run 'crc setup' again.",
```

Comment:
The commit log says a re-login is sometimes necessary, but this removes it from the message? Do we need to run crc setup again after rebooting to configure the virtual switch?

Reply:
>The commit log says a re-login is sometimes necessary, but this removes it from the message?

Ahh, need to modify the commit msg then, it should also say to reboot, i've had less luck wiht re-login, but reboot always seems to work.

>Do we need to run crc setup again after rebooting to configure the virtual switch?

user doesn't have to run `crc setup` again, but i added this so that user knows if all the setup steps were successfully done.

How about asking to run `crc start`, `Unable to perform Hyper-V administrative commands. Please reboot your system and run 'crc start'.` as the same preflight cheks will be done for start also.

Reply:
I did not  follow you :) If after a reboot, `setup` is never required, the message can just be `Please reboot your system to complete the setup process`. If `crc setup` is needed in some cases, then the current message is good in these cases :)

Reply:
Changed the message to `Please reboot your system to complete the setup process`.

I had the weird idea of making the users run `crc setup` again just to verify if the setup succeeded earlier ;-)